Client/Assets/NightModeHelper.js

```diff
@@ -0,0 +1,44 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+(function() {
+  if (!window.__firefox__) {
+    window.__firefox__ = {};
+  }
+  var isEnabled = false;
+
+  function initializeStyleSheet (enabled) {
+    var night_css_id = "iBrowser_night_css";
+    var night_css = ':not(.UIButton){color: #AAAAAA !important; background-color:#2E2E2E !important;}body{-webkit-filter:brightness(.5) !important;}';
```

Comment:
Taobao.com is the most popular website in China, and they use the class UIButton to display their content. If we change the background-color on UIButton, the content will be covered. UIButton is just used in taobao's website, so we set a special rule on it.
Because the website is different between China and outside China, I need go back to Beijing to reply the detail for you.